concent_api/core/validation.py

```diff
@@ -69,3 +69,10 @@ def validate_golem_message_task_to_compute(golem_message: message.base.Message):
 
     validate_public_key(golem_message.provider_public_key, 'provider_public_key')
     validate_public_key(golem_message.requestor_public_key, 'requestor_public_key')
+
+
+def validate_report_computed_task_time_window(report_computed_task):
```

Comment:
please make unit tests for it

Reply:
`test_send_should_return_http_400_if_task_to_compute_younger_than_report_computed` in `test_core_views.py` is to check that method. It's not enough?

Reply:
Ok, we will talk about that on the next Tech Talk ;)